sys/include/net/nanocoap.h

```diff
@@ -1248,6 +1264,23 @@ static inline size_t coap_opt_put_uri_query(uint8_t *buf, uint16_t lastonum,
     return coap_opt_put_string(buf, lastonum, COAP_OPT_URI_QUERY, uri, '&');
 }
 
+/**
+ * @brief   Convenience function for inserting PROXY_URI option into buffer
+ *
+ * @param[out]  buf         buffer to write to
+ * @param[in]   lastonum    number of previous option (for delta calculation),
+ *                          or 0 if first option
+ * @param[in]   uri         ptr to source URI
+ *
+ * @returns     amount of bytes written to @p buf
+ */
+static inline size_t coap_opt_put_proxy_uri(uint8_t *buf, uint16_t lastonum,
+                                            const char *uri)
+{
+    return coap_opt_put_string(buf, lastonum, COAP_OPT_URI_PATH, uri, '\0');
```

Comment:
I think you meant to use COAP_OPT_PROXY_URI. I obviously didn't test this. :-/

Please squash in directly.

Reply:
squashed. The `put` variant is not used anyway (the `add` variant is used). So your tests could never have caught that :/